src/opera/commands/deploy.py

```diff
@@ -57,3 +73,30 @@ def deploy(args):
         return 1
 
     return 0
+
+
+def deploy_compressed_csar(args, storage):
+    with TemporaryDirectory(dir=storage.path) as tempdir:
+        ZipFile(args.csar.name, 'r').extractall(tempdir)
+
+        try:
+            inputs_file = open(tempdir + "/" + args.inputs, 'r')
```

Comment:
Inputs shouldn't be packaged into the CSAR. They are a separate file.

Reply:
Ok, so now the inputs are left out of CSARs and provided optionally as a separate file.